### PR TITLE
feat: Add content to blips Part 2

### PR DESCRIPTION
**Pull request by andrablaj**
Part 2 of updating blips content.

### CODE REVIEW

File: radar/2023-09-12/angularjs.md [hunk 1]
```diff
@@ -2,7 +2,12 @@
...
 ---
 
-Obsolete version of Angular, used in the CHT admin app. Deprecated: rewrite to Angular.
\ No newline at end of file
+[AngularJS](https://angularjs.org/) is an obsolete version of Angular, used in the CHT admin app. Deprecated: rewrite to Angular.
+
+The CHT UI framework was upgraded from the deprecated AngularJS v1.6 to Angular in the 
+[version 3.11](https://docs.communityhealthtoolkit.org/core/releases/3.11.0/). 
+The upgrade drastically reduces the memory used on the phone. 
+It makes development of the Core Framework easier and more reliable and also makes it easier to keep on the latest version of Angular which means we can keep up to date and on a supported version.
```
Comment (garethbowen) [inline radar/2023-09-12/angularjs.md]: It's worth noting that AngularJS is still in use in the "admin" app.

File: radar/2023-09-12/github-actions.md [hunk 2]
```diff
@@ -5,4 +5,4 @@ quadrant:   tools
 tags:       [infrastructure]
 ---
 
-Used extensively for CI across CHT products including testing and releasing.
\ No newline at end of file
+[GitHub Actions](https://github.com/features/actions) is used extensively for Continuous Integration (CI) across CHT products including testing and releasing.
```
Comment (garethbowen) [inline radar/2023-09-12/github-actions.md:8]: Do you think we should add "GitHub" as well?

File: radar/2023-09-12/grafana.md [hunk 3]
```diff
@@ -5,4 +5,4 @@ quadrant:   tools
 tags:       [monitoring, alerting]
 ---
 
-Being evaluated for CHT Watchdog.
\ No newline at end of file
+Grafana is being evaluated for CHT Watchdog.
```
Comment (garethbowen) [inline radar/2023-09-12/grafana.md:8]: I think we can promote this from "assess" to "adopt" now. @mrjones-plip What do you think?

File: radar/2023-09-12/kubernetes.md [hunk 4]
```diff
@@ -5,4 +5,5 @@ quadrant:   tools
 tags:       [infrastructure]
 ---
 
-Only needed for Medic hosting right now.
\ No newline at end of file
+Kubernetes is only needed for Medic hosting right now.
```
Comment (garethbowen) [inline radar/2023-09-12/kubernetes.md]: We're promoting k3s/k3d for self hosting too, which is k8s compatible, so I think it'd be better to say...

```suggestion
Kubernetes platform is used for hosting the CHT. For self hosting partners we recommend lightweight implementations like k3d. For cloud deployments, use a Platform as a Service offering like Amazon EKS.
```

Might still be too technical but you get the idea...

File: radar/2023-09-12/pouchdb.md [hunk 5]
```diff
@@ -5,5 +5,7 @@ quadrant:   languages-and-frameworks
 tags:       [framework]
 ---
 
-JavaScript library for storing data on the phone in the CHT webapp. 
-Also used as a library for querying CouchDB from api, sentinel, and the admin app.
+[PouchDB](https://pouchdb.com/) is an open-source JavaScript database inspired by Apache CouchDB that is designed to run well within the browser.
```
Comment (garethbowen) [inline radar/2023-09-12/pouchdb.md]:
```suggestion
[PouchDB](https://pouchdb.com/) is an open-source JavaScript database inspired by Apache CouchDB that is designed to run within the browser.
```

File: radar/2023-09-12/pouchdb.md [hunk 6]
```diff
@@ -5,5 +5,7 @@ quadrant:   languages-and-frameworks
...
+
+PouchDB is used for storing data on the phone in the CHT webapp.
```
Comment (garethbowen) [inline radar/2023-09-12/pouchdb.md]:
```suggestion
PouchDB is used for storing data on the phone in the CHT webapp to enable it to work offline seamlessly. 
```

File: radar/2023-09-12/pwa.md [hunk 7]
```diff
@@ -5,4 +5,7 @@ quadrant:   methods-and-patterns
 tags:       [technique]
 ---
 
-Followed for CHT webapp to allow offline first capability.
\ No newline at end of file
+A progressive web application (PWA), or progressive web app, is a type of application software delivered through the web, built using common web technologies including HTML, CSS, and JavaScript.
```
Comment (garethbowen) [inline radar/2023-09-12/pwa.md]:
```suggestion
A progressive web application (PWA) is a type of application software delivered through the web, built using common web technologies including HTML, CSS, and JavaScript. 
```

File: radar/2023-09-13/amazon-eks.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+title:      "Amazon EKS"
+ring:       adopt
+quadrant:   platforms-and-aoe-services
+tags:       [infrastructure]
+---
+
+[Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/de/eks/) is a managed service that makes it easy to run Kubernetes on AWS.
+It is currently used fore Medic hosted CHT deployments.
```
Comment (garethbowen) [inline radar/2023-09-13/amazon-eks.md]:
```suggestion
It is currently used for Medic hosted CHT deployments.
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice one!

File: radar/2023-09-12/grafana.md (hunk 3 above)
Comment (mrjones-plip) [reply radar/2023-09-12/grafana.md:8]: yup yup - we're all in!

File: radar/2023-09-12/github-actions.md (hunk 2 above)
Comment (andrablaj) [reply radar/2023-09-12/github-actions.md:8]: Great idea - done!

File: radar/2023-09-12/grafana.md (hunk 3 above)
Comment (andrablaj) [reply radar/2023-09-12/grafana.md:8]: There are two blips on Grafana. This one where you commented is the initial version when we created the radar. The second blip can be found [here](https://github.com/medic/cht-tech-radar-contributors/blob/main/radar/2023-09-13/grafana.md), and it contains the updated status (adopt), plus extra info on Grafana. The radar merges the info from the two blips, and shows the newest one first. Let me know what you think.

Comment (garethbowen) [reply radar/2023-09-12/grafana.md:8]: Of course. That's not obvious in the PR review UI  :/

Thanks for explaining.

**Review by garethbowen — APPROVED** [review]
Excellent. Thanks!

**Comment by andrablaj** [comment]
Oh, sorry, I forgot to squash my commits. 😔

**Pull request merged**